fuzzbench/test_e2e_run.py

```diff
@@ -18,17 +18,38 @@
 import os
 
 import pytest
+import redis
+from rq.job import Job
+
+
+@pytest.fixture(scope='module')
```

Comment:
Not sure which scope level should be used here.
function or class or module or nothing needed?

Reply:
i'd use `class` and use it with `TestEndToEndRunResults`

Reply:
Sounds good! Done.